packages/content-management/content_insights/content_insights_public/src/components/views_stats/views_stats.tsx

```diff
@@ -0,0 +1,134 @@
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useQuery } from '@tanstack/react-query';
+import { i18n } from '@kbn/i18n';
+import type { ContentInsightsStats } from '@kbn/content-management-content-insights-server';
+import { css } from '@emotion/react';
+import moment from 'moment';
+
+import { Item } from '../../types';
+import { ViewsChart } from './views_chart';
+import { useServices } from '../../services';
+
+export const ViewsStats = ({ item }: { item: Item }) => {
+  const contentInsightsClient = useServices()?.contentInsightsClient;
+
+  if (!contentInsightsClient) {
+    throw new Error('Content insights client is not available');
+  }
+
+  const { euiTheme } = useEuiTheme();
+  const { data, isLoading } = useQuery(
+    ['content-insights:viewed', item.id],
+    async () =>
+      contentInsightsClient.getStats(item.id, 'viewed').then((response) => ({
+        totalDays: getTotalDays(response),
+        totalViews: getTotalViews(response),
+        chartData: getChartData(response),
+      })),
+    {
+      staleTime: 0,
+      retry: false,
+    }
+  );
+
+  return (
+    <EuiPanel hasBorder paddingSize={'s'}>
+      <EuiStat
+        data-test-subj={'views-stats-total-views'}
+        title={data?.totalViews ?? '–'}
+        description={
+          <>
+            <FormattedMessage
+              id="contentManagement.contentEditor.viewsStats.viewsLastNDaysLabel"
+              defaultMessage="Views (last {n} days)"
+              values={{ n: data?.totalDays }}
+            />
+            <NoViewsTip />
+          </>
+        }
+        isLoading={isLoading}
+      />
+      <EuiSpacer
+        size={'s'}
+        css={css`
+          border-bottom: ${euiTheme.border.thin};
+        `}
+      />
+      <EuiSpacer size={'m'} />
+
+      <ViewsChart data={data?.chartData ?? []} />
+    </EuiPanel>
+  );
+};
+
+const NoViewsTip = () => (
+  <EuiIconTip
+    aria-label={i18n.translate('contentManagement.contentEditor.viewsStats.noViewsTipAriaLabel', {
+      defaultMessage: 'Additional information',
+    })}
+    position="top"
+    color="inherit"
+    iconProps={{ style: { verticalAlign: 'text-bottom', marginLeft: 2 } }}
+    css={{ textWrap: 'balance' }}
+    type="questionInCircle"
+    content={
+      <FormattedMessage
+        id="contentManagement.contentEditor.viewsStats.noViewsTip"
+        defaultMessage="Views are counted everytime someone opens a dashboard (after version 8.16)"
+      />
+    }
+  />
+);
+
+function getTotalViews(stats: ContentInsightsStats) {
+  return stats.daily.reduce((acc, { count }) => acc + count, 0);
+}
+
+export function getTotalDays(stats: ContentInsightsStats) {
+  const from = new Date(stats.from);
+  const currentDate = new Date();
+  const currentUTCDate = new Date(
+    Date.UTC(currentDate.getUTCFullYear(), currentDate.getUTCMonth(), currentDate.getUTCDate())
+  );
+
+  return Math.round((currentUTCDate.getTime() - from.getTime()) / (1000 * 60 * 60 * 24));
```

Comment:
nit: while `moment` is already in place you can use its helpers

```suggestion
  return moment.utc().diff(moment.utc(stats.from), 'days');
```